client/client.go

```diff
@@ -15,15 +15,15 @@ const (
 	EngineTypeSQLServer = "SQLSERVER"
 )
 
-// Instance is the api message of the instance.
-type Instance struct {
-	ID                 string
-	ServiceCode        string     `json:"servicecode"`
-	Region             string     `json:"region"`
-	Type               string     `json:"instanceType"`
-	InstanceFamily     string     `json:"instanceFamily"`
-	VCPU               string     `json:"vcpu"`
-	Memory             string     `json:"memory"`
+// OfferPayloadInstance is the payload of the offer type instance.
+type OfferPayloadInstance struct {
+	// e.g. db.lg, N1Standard-1-1
+	Type string `json:"instanceType"`
+	// e.g. HighMem, Generals
+	InstanceFamily string `json:"instanceFamily"`
+	VCPU           string `json:"vcpu"`
```

Comment:
here, we still say vCPU, while the early place changed to CPU. What's the thought behind the change.

Reply:
vCPU is part of a instance, while CPU can be offered independently. 
But I think both naming is ok. Here I just follows GCP's.